pkg/importer/tests-openapi/nested-types.sysl

```diff
@@ -21,38 +21,29 @@ testapp "Fruit API" [package="package_foo"]:
             @json_tag = "veggies"
 
     !type Details:
-        month <: Details_month?:
+        month <: string?:
```

Comment:
probably out of scope but looking at the openapi spec, it seems like this is supposed to be an enum https://github.com/anz-bank/sysl/blob/master/pkg/importer/tests-openapi/nested-types.yaml#L59

Reply:
correct, there is a `if 0`'d out block of code which I'll reenable very soon which adds sysl attributes to these strings to add the legal values

Reply:
there are enums in in sysl https://github.com/anz-bank/sysl/blob/master/pkg/parse/tests/enum.sysl but looking at the enums in the openapi spec, maybe sysl enum won't work as well (enums in sysl uses numbers).